app/jobs/split_pdf_job.rb

```diff
@@ -258,10 +258,9 @@ def save_pages(exam_template, partial_exams, filename = nil, split_pdf_log = nil
         img.destroy!
         student_info.destroy!
 
-        python_exe = Rails.application.config.python
-        read_chars_py_file = Rails.root.join('lib/scanner/run_scanner.py').to_s
+        exam_matcher_exe = "#{Rails.application.config.python} -m markus_exam_matcher"
         char_type = exam_template.cover_fields == 'id_number' ? 'digit' : 'letter'
-        stdout, status = Open3.capture2(python_exe, read_chars_py_file, student_info_file, char_type)
+        stdout, status = Open3.capture2("#{exam_matcher_exe} #{student_info_file} #{char_type}")
```

Comment:
In general command line arguments should be passed individually (like it was before), not as a concatenated string. This allows the function to correctly quote the arguments (e.g. when there's spaces in filenames) so that you don't need to worry about doing so.

Reply:
This is definitely a better solution, thanks! In case it is important, the code that runs `pip freeze` in `config/initializers/python.rb` is written like so:
```
pip = "#{Rails.application.config.python} -m pip"

begin
  installed, status = Open3.capture2("#{pip} freeze")
  ...
```
Maybe this should also be changed for consistency?

Reply:
Ah yes, you can change that as well. (It matters less because `Rails.application.config.python` isn't a user-provided input, but still good to follow best practice.)

Reply:
Done!